packages/cdr/docker/jsreport/scripts/report-5f/content.js

```diff
@@ -0,0 +1,186 @@
+const axios = require('axios')
+
+const ES_INDEX = 'fhir-report-patient'
+const ES_USERNAME = 'elastic'
+const ES_PASSWORD = 'dev_password_only'
+
+async function beforeRender(req) {
+  const { from, to, state, district, city, facilityCode } = req.data.params
+
+  const esQuery = {
+    size: 0,
+    query: {
+      bool: {
+        filter: [
+          {
+            range: {
+              'artInitiation.dateInitiated': {
+                gte: `${from}||/d`,
+                lte: `${to}||/d`
+              }
+            }
+          },
+          {
+            exists: {
+              field: 'hivDiagnosis.hivPosDate'
+            }
+          },
+          {
+            exists: {
+              field: 'artInitiation.dateInitiated'
+            }
+          },
```

Comment:
This is redundant as you already are using this field in the range query above.

Reply:
@rcrichton done